package.json

```diff
@@ -17,8 +18,8 @@
     "fix:js": "xo --fix ./src",
     "fix:sass": "sass-lint --fix ./src",
     "fix:vulnerabilities": "snyk protect",
-    "start": "if [ \"$NODE_ENV\" = 'production' ]; then npm run start:production; else npm run start:dev; fi",
-    "start:dev": "NODE_ENV=development ENABLE_WATCH=true ENABLE_SERVER=true npm run build | pino",
+    "start": "export PORT=${PORT-3000}; if [ \"$NODE_ENV\" = 'production' ]; then npm run -s start:production; else npm run -s start:dev; fi",
```

Comment:
Shouldn't it be `${PORT:-3000}`? 🤔 

It's getting a little complicated. I wonder when are we going to say "let's use Makefile" :D

Reply:
Both are valid in this case ... I think... `${PORT-3000}` will use 3000 if PORT is 'unset' ... which allows setting it to nothing `PORT=` to get the random port behaviour if you were so inclined

Reply:
TIL.